roles/pytest_project/defaults/main.yml

```diff
@@ -19,3 +19,5 @@ pytest_project_junit_output: junit.xml
 pytest_project_markers:
 pytest_project_command_args:
 pytest_project_ignore_errors: no
+pytest_local_env: false
+pytest_project_alias: "pytest-project"
```

Comment:
Perhaps this should support an empty string to skip setting up the alias. Perhaps that should even be the default